README.md

```diff
@@ -217,11 +217,12 @@ Please refer to the respective documentation for further details to manage packa
 ### Activate shell completions for nf-core/tools
 
 Auto-completion for the `nf-core` command is available for bash, zsh and fish. To activate it, add the following lines to the respective shell config files.
-shell | shell config file | command
---- | --- | ---
-bash | ~/.bashrc | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`
-Zsh | ~/.zshrc | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`
-fish | ~/.config/fish/completions/nf-core.fish | `eval (env _NF_CORE_COMPLETE=fish_source nf-core)`
+
+| shell | shell config file                       | command                                            |
+| ----- | --------------------------------------- | -------------------------------------------------- |
+| bash  | ~/.bashrc                               | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`  |
+| Zsh   | ~/.zshrc                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
```

Comment:
Lower case?

```suggestion
| zsh   | ~/.zshrc                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
```

Reply:
the authors don't seem to be consistent about it,think I prefer lower case, but I was following how Wikipedia writes it 🤷🏻